gateway/sds_gateway/api_methods/helpers/file_helpers.py

```diff
@@ -0,0 +1,74 @@
+from rest_framework import status
+from rest_framework.parsers import MultiPartParser
+from rest_framework.request import Request
+from rest_framework.test import APIRequestFactory
+
+from sds_gateway.api_methods.views.file_endpoints import CheckFileContentsExistView
+from sds_gateway.api_methods.views.file_endpoints import FileViewSet
+
+
+def upload_file_helper_simple(request, file_data):
+    """
+    Upload a single file using FileViewSet.create.
+    file_data should contain all required fields: name, directory, file,
+    media_type, etc.
+    Returns ([response], []) for success, ([], [error]) for error, and handles
+    409 as a warning.
+    """
```

Comment:
[nit] try to follow this style guide for docstrings; you'll see it in other methods too

https://google.github.io/styleguide/pyguide.html#383-functions-and-methods